build.gradle.kts

```diff
@@ -2,4 +2,44 @@ plugins {
     kotlin("jvm") version "1.7.22" apply false
     id("com.squareup.sqldelight") version "1.5.4" apply false
     id("io.gitlab.arturbosch.detekt") version "1.22.0" apply false
+    id("com.diffplug.spotless") version "6.12.0"
+}
+
+buildscript {
+    dependencies {
+        classpath("com.pinterest:ktlint:0.47.1")
+    }
+}
+
+subprojects {
+    apply(plugin = "com.diffplug.spotless")
```

Comment:
What do you think about using IntelliJ copyright notice features to handle this instead of Spotless? We can catch missing values (if any) with Detekt

Reply:
For reference, configuration of this is checked in to VCS, and it seems to apply to new files created by default

Reply:
Done, check it out